packages/caliper-ethereum/package.json

```diff
@@ -26,7 +26,7 @@
         "ethereumjs-wallet": "^0.6.3"
     },
     "devDependencies": {
-        "web3": "1.2.2",
+        "web3": "0.20.7",
```

Comment:
@aklenik This change broke the ethereum adapter. I'm assuming you made this change to quiet lerna's warning about differing web3 versions between the burrow and ethereum adapters? If so, can I please suggest that the burrow adapter be updated to depend on v1.2.2? Version 0.20.7 is quite old now, and it doesn't support websockets, subscriptions, etc.

Reply:
I've submitted #825 to revert this.

Reply:
Good catch! The old web3 is needed by the fisco adapter, not burrow. Hopefully, this only caused dev-time issues, since the binding specification contains the correct version.